essentials/src/utils.rs

```diff
@@ -15,17 +15,30 @@
 // along with polkadot-introspector.  If not, see <http://www.gnu.org/licenses/>.
 //
 
-use std::time::Duration;
-
+use clap::Parser;
 use log::info;
+use std::time::Duration;
 use thiserror::Error;
 use tokio::time::sleep;
 
-use crate::constants::{RETRY_COUNT, RETRY_DELAY_MS};
+#[derive(Clone, Debug, Parser)]
+#[clap(
+	rename_all = "kebab-case",
+	allow_external_subcommands = true, // HACK: to parse it as a standalone config
+)]
+pub struct RetryOptions {
+	/// Max of times to retry a failed connection before giving up.
+	#[clap(name = "retry", default_value = "3", long)]
+	max_count: u32,
+	/// Delay in ms to wait between retry attempts
+	#[clap(default_value = "100", long)]
+	retry_delay: u32,
```

Comment:
I usually prefer time intervals expressed in seconds with a floating point. But it might be probably ok to use milliseconds here.

Reply:
We need to answer the topic starter :-)
@sandreim what do you think?

Reply:
I have no personal preference in this case. One downside of floating point numbers is that they are approximate and we should refrain from ever using them in this industry even for tools.

Reply:
So let's leave ms here.